### ISSUE TITLE
Gestures Broken in 3.2 Pre-Release

### ISSUE THREAD

**Issue by AdamRoden** [OptiKey/OptiKey#800]
I'm not able to activate individual gestures. It is all or nothing. Also, they can no longer be tested on the fly.

**Comment by JuliusSweetland** [comment]
Has something broken your code or have you just noticed a bug?

**Comment by AdamRoden** [comment]
When @kmcnaught was looking into things I suspect one of the bug fixes to gestures affected more than expected.

**Comment by JuliusSweetland** [comment]
@AdamRoden @kmcnaught @annakirkpatrick So it is my understanding that this is the only remaining bug in 3.2. Please let me know if there are others.

**Comment by AdamRoden** [comment]
@JuliusSweetland I just submit a PR to fix.

**Comment by JuliusSweetland** [comment]
@AdamRoden Merged. Nice one Adam. I'll prep another pre-release of 3.2 now.

**Comment by JuliusSweetland** [comment]
@kmcnaught @AdamRoden Latest version here: https://github.com/OptiKey/OptiKey/releases/tag/v3.2.0_pre-release-3

Looking good for a general release?

**Comment by AdamRoden** [comment]
No. :-( Now the gaze indicator gets coerced so it is always 100% on the screen.

**Comment by JuliusSweetland** [comment]
@AdamRoden Does #804 (merged) fix everything related to this issue? I'm really asking if I should wait for anything else before preparing a new pre-release? Thanks.
